.github/workflows/basics.yml

```diff
@@ -40,9 +39,22 @@ jobs:
     - run: yarn --frozen-lockfile
     - run: yarn build-all
     - run: yarn diff:sample-json
+    - run: yarn type-check
     - run: yarn lint
     - run: yarn unit-core
```

Comment:
also need `unit-cli` and `unit-viewer`

i added a note that these will move to their coverage eequivalents later.